Tools/CISupport/ews-build/events.py

```diff
@@ -278,6 +280,63 @@ def __exit__(self, type, value, tb):
         log.msg = self.saved_logger
 
 
+# Based off of webkitscmpy's CommitClassifier, although modified to not raise exceptions
+# when provided invalid commit classes.
+class CommitClass(object):
```

Comment:
The name can be more descriptive to indicate what's the purpose of this class, e.g.: CommitClassifier